Tasks/AzurePowerShell/AzurePowerShell.ps1

```diff
@@ -70,6 +70,11 @@ catch
    Write-Verbose "Unable to get the authScheme $error" 
 }
 
+if ($endpoint.Auth.parameters.authenticationType -ne $NULL -and $endpoint.Auth.parameters.authenticationType -eq "spnCertificate")
```

Comment:
Shouldn't the condition after -and suffice ? Why the null check ?

Reply:
removed the first condition. thanks